source/PyMaterialX/PyXmlIo.cpp

```diff
@@ -16,7 +16,7 @@ namespace mx = MaterialX;
 void bindPyXmlIo(py::module& mod)
 {
     mod.def("readFromXmlFileBase", &mx::readFromXmlFile,
-        py::arg("doc"), py::arg("filename"), py::arg("searchPath") = mx::EMPTY_STRING, py::arg("readXIncludes") = true);
+        py::arg("doc"), py::arg("filename"), py::arg("searchPath") = mx::EMPTY_STRING, py::arg("readingOptions") = static_cast<mx::XmlReadOptions*>(nullptr));
```

Comment:
Maybe this needs to be a cast to a (const XmlReadOptions*)?  PyBind11 has very strict requirements about how default arguments are declared, and the missing "const" might cause this to break.

http://pybind11.readthedocs.io/en/stable/advanced/functions.html#default-arguments-revisited